services_app/src/androidTest/java/org/opendatakit/database/service/OdkDatabaseServiceTest.java

```diff
@@ -308,7 +312,8 @@ public void testDbInsertThrowsIllegalArgumentException()
          List<Column> columnList = createColumnList();
          ColumnList colList = new ColumnList(columnList);
 
-         db = serviceInterface.openDatabase(APPNAME);
+          assertNotNull(serviceInterface);
```

Comment:
why did you add an assertNotNull for serviceInterface but not for DBHandle? Seem like we should be consistent. Maybe it's impossible for DbHandle to be null (you might want to check on that)